src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Tests/ITestTemplate.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using WcfService1;
+
+namespace SharedPoolsOfWCFObjects
+{
+    public interface ITestTemplate<C>
+    {
```

Comment:
Nit: in general, generic parameters need better names, like TClient or something.  Not required for merge.